docs/configuration/OVERVIEW.md

```diff
@@ -1,31 +1,31 @@
 # Configuration mechanism
 
-The aim of this document is to help developers to implement configuration inside components, in order to be compliant
-with __cms architecture__. This will give the possibility to __Business Analyst__ to configure the
+The aim of this document is to help developers to implement `Configuration` inside components, in order to be compliant with the __CMS architecture__.
```

Comment:
I think the first thing is not about the CMS, it's about making the application configurable dynamically. The CMS is a way to ease the creation of this configuration override, but that's not mandatory. It would be interesting to develop "When do I want to build configurability in my application and why" and then go on the How which is already part of this file